Gemfile.lock

```diff
@@ -318,6 +318,3 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   unicorn (~> 4.9.0)
   webmock (~> 1.21.0)
-
-BUNDLED WITH
-   1.10.6
```

Comment:
I don't think you meant to commit this change... (collections is now on ruby 2.2.3, which uses bundler 1.10.6 in our stack)